src/CMakeLists.txt

```diff
@@ -153,10 +153,10 @@ endif()
 if(ONEDNN_BUILD_GRAPH)
     message(STATUS "Graph component is enabled")
 
-    if (NOT DNNL_GPU_RUNTIME STREQUAL "NONE" AND NOT DNNL_GPU_VENDOR STREQUAL "INTEL")
+    if (NOT DNNL_GPU_RUNTIME STREQUAL "NONE" AND DNNL_GPU_VENDOR STREQUAL "AMD")
         message(FATAL_ERROR "Graph API does not support ${DNNL_GPU_VENDOR} GPU. "
             "Either disable Graph API with ONEDNN_BUILD_GRAPH=OFF or change GPU "
-            "vendor to INTEL with ONEDNN_GPU_VENDOR=INTEL.")
+            "vendor to INTEL or NVIDIA.")
```

Comment:
Can you clarify, are the changes in this PR enough to use oneDNN graph API with the native CUDA backend? Does it end up using the existing implementation in https://github.com/oneapi-src/oneDNN/tree/main/src/gpu/nvidia?
I imagine this won't be enough to build and execute a graph like a CUDA graph API would, is this the first PR of a series to come to extend the CUDA Graph capabilities in oneDNN?

Reply:
Yes sorry I meant cuDNN Graph API. Some users may expect that using oneDNN Graph API with an Nvidia device should provide similar functionalities and performance improvements than using one of cuDNN Graph API (backend or frontend).
I understand it would be a lot of work to support one of the cuDNN Graph APIs. Thanks for your answer.